customers.sql

```diff
@@ -0,0 +1,21 @@
+with customer_orders as (
+  select
+    customer_id,
+    count(*) as n_orders,
+    min(created_at) as first_order_at,
+  from
+    `analytics-engineers-club.coffee_shop.orders`
+  group by
+    customer_id
+
+)
+
+select 
+  customers.id as customer_id,
+  customers.name,
+  customers.email,
+  coalesce(customer_orders.n_orders,0) as number_of_orders,
+  customer_orders.first_order_at
+from `analytics-engineers-club.coffee_shop.customers` customers
+left join customer_orders
+  on customers.id = customer_orders.customer_id
```

Comment:
You might be aware of this already, but if you want to remove this little newline warning from GitHub you can just create an empty new line at the end of your file. It doesn't matter for `.sql` files afaik, but I think it might matter for other languages. 😄

Reply:
oooh I did not know that. Thanks Katie! added a new commit to fix that